__tests__/integration/multisig.test.js

```diff
@@ -267,4 +267,35 @@ describe('send tx (HTR)', () => {
     expect(response.body.success).toBe(false);
     expect(response.body.error).toBeDefined();
   });
+
+  it('should mount the transaction with the correct change outputs', async () => {
```

Comment:
Do we have a test that we use `tx-proposal` with the `change_address` parameter filled?

Reply:
Yes, and we test when the change address belongs to the wallet and that it fails when it does not belong to the wallet.